src/components/TheNavBar.vue

```diff
@@ -123,6 +131,58 @@
       </div>
     </div>
   </nav>
+  <div class="modal is-active" v-if="showAboutModal">
+    <div class="modal-background"></div>
+    <div class="modal-card">
+      <header class="modal-card-head">
+        <p class="modal-card-title is-title-shrink">
+          {{ $gettext("About ") }}{{ this.appName }}
+        </p>
+        <button
+          class="delete"
+          aria-label="close"
+          @click="
+            ;(showAboutModal = false), $store.commit('setModalState', false)
+          "
+        ></button>
+      </header>
+
+      <section class="modal-card-body">
+        <div>
+          <div v-if="$config.logoUrl" class="app-version mb-4">
+            <img class="app-img" :src="$config.logoUrl" />
+          </div>
+          <div class="app-version mb-3">
+            <span class="mb-2">
+              <strong>{{ this.appName }}</strong> {{ $gettext("version ") }}
+            </span>
+            <span>
+              <strong>{{ this.appVersion }}</strong>
+            </span>
+          </div>
+          <div class="has-text-centered">
+            <strong>{{ this.appName }} </strong>
+            {{ $gettext("is designed by") }}
+            <a href="https://lokavaluto.fr/" target="_blank">
+              <strong>lokavaluto </strong>
+            </a>
+            {{
+              $gettext(
+                " an association whose goal is to develop, implement, and support the deployment of evolving and open-source information systems for organizations promoting local economic development."
```

Comment:
Can you ask a feedback to stephane for the text and the general behavior by setting up a version on `dev3` ?